### PR TITLE
[5.0] Drop tinyMCE own languages

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/40408 .

### Summary of Changes

Use the official translation files

### Testing Instructions

Run `npm ci` and check that changing the language in tinyMCE still works as before

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

@tecpromotion @brianteeman

A note here: this could be alternative done as a GitHub action similar to the Joomla languages with the benefit that there will not be an additional nom dependency for the `adm-zip` module and also keep the tools execution time shorter...

### CODE REVIEW

File: build/build-modules-js/init/minify-vendor.es6.js [hunk 1]
```diff
@@ -55,7 +56,7 @@ const minifyJS = async (file) => {
   }
 
   // eslint-disable-next-line no-console
-  console.log(`Processing Vendor file: ${file}`);
+  // console.log(`Processing Vendor file: ${file}`);
```
Comment (brianteeman) [inline build/build-modules-js/init/minify-vendor.es6.js]: if this line is commented out then surely the line above is not needed

**Comment by dgrammatiko** [comment]
@HLeithner @Hackwar can someone approve the node version bump to v18 on the drone for v5?

FWIW v16 (currently used) will be EOL by the time v5 will be released: https://github.com/nodejs/Release/blob/main/schedule.svg

**Comment by dgrammatiko** [comment]
Sure, I wanted to see what's the real impact on the `npm ci` with the additional fetch/unzip/minification of the files...

**Comment by laoneo** [comment]
Should we not load it through npm https://github.com/mklkj/tinymce-i18n?

**Comment by dgrammatiko** [comment]
> Should we not load it through npm https://github.com/mklkj/tinymce-i18n?

Personally I'm against depending on a random developer for something that essentially is done in the same way: https://github.com/mklkj/tinymce-i18n/blob/a62cd4a29fe10d85aaefe6062abd10fc6cbfd938/update.sh#L3

FWIW maybe you can create a tinyMCE-languages under Joomla-projects and I can create a similar automation (the point here is security, ownership, bus factor, etc). But if people are ok with pulling random packages then I can change the nom-tools to use that package...

File: build/build-modules-js/init/exemptions/tinymce.es6.js [hunk 2]
```diff
@@ -90,6 +90,20 @@ module.exports.tinyMCE = async (packageName, version) => {
 </html>
 `;
 
+  // Get the languages
+  await fetch('https://download.tiny.cloud/tinymce/community/languagepacks/6/langs.zip')
```
Comment (Fedik) [inline build/build-modules-js/init/exemptions/tinymce.es6.js:94]: Can we cache it somehow?
Downloading 400Kb+ on each npm run, does not sounds fun :)
I mean even npm have its own cache for that reason.

Comment (dgrammatiko) [reply build/build-modules-js/init/exemptions/tinymce.es6.js:94]: @Fedik probably this portion of the code would be moved to the https://github.com/joomla/core-translations as a Github action (cron based I guess) so the tools (finally) won't have any perf penalty or a need for caching (the files would automatically merged on this branch per the existing action)

**Pull request closed without merging**